auth/client/src/main/java/org/wildfly/security/auth/client/ElytronXmlParser.java

```diff
@@ -678,7 +724,7 @@ private static void parseCertificateRevocationList(ConfigurationXMLStreamReader
                     break;
                 }
                 case "maximum-cert-path": {
-                    if (maxCertPath != 0) throw reader.unexpectedAttribute(i);
+                    if (maxCertPath != 0 || xmlVersion.isAtLeast(Version.VERSION_1_4)) throw reader.unexpectedAttribute(i);
```

Comment:
Same here, for backwards compatibility, ```maximum-cert-path``` still needs to be allowed here. It would be good to log a message indicating it has been deprecated though.

Reply:
fixed :)